src/main/java/Duke.java

```diff
@@ -1,10 +1,179 @@
+                System.out.println(userTasks[taskIndex]);
+                break;
+
+            case "unmark":
+                System.out.println("OK, I've marked this task as not done yet:");
+                taskIndex = Integer.parseInt(userCommands[1]) - 1;
+                userTasks[taskIndex].setisDone(false);
+                //System.out.println("  [ ] " + userTasks[taskIndex].getTaskName());
+                System.out.println(userTasks[taskIndex]);
+                break;
+
+            case "todo":
+                System.out.println("Got it. I've added this task:"); // shift this line below with the another print statement later
+                String taskString = GetTaskString(userInput,userCommand);
+                String taskName = GetToDoTaskName(taskString);
+                Todo newToDoTask = new Todo(taskName);
+                userTasks = AddUserTask(userTasks, newToDoTask);
+                System.out.println(userTasks[userTasks.length-1]);
+                System.out.println("Now you have " + userTasks.length + " in the list.");
+                break;
+
+
+            case "deadline":
+                System.out.println("Got it. I've added this task:"); // shift this line below with the another print statement later
+                taskString = GetTaskString(userInput,userCommand);
+                taskName = GetDeadlineTaskName(taskString);
+
+                String deadlineDueDate = GetDeadlineDueDateString(taskString);
+                Deadline newDeadlineTask = new Deadline(taskName,deadlineDueDate);
+                userTasks = AddUserTask(userTasks, newDeadlineTask);
+                System.out.println(userTasks[userTasks.length-1]);
+                System.out.println("Now you have " + userTasks.length + " in the list.");
+                break;
+
+            case "event":
+                System.out.println("Got it. I've added this task:"); // shift this line below with the another print statement later
+                taskString = GetTaskString(userInput,userCommand);
+                taskName = GetEventTaskName(taskString);
+
+                String eventFromDate = GetEventFromDate(taskString);
+                String eventToDate = GetEventToDate(taskString);
+                Event newEventTask = new Event(taskName,eventFromDate,eventToDate);
+                userTasks = AddUserTask(userTasks, newEventTask);
+                System.out.println(userTasks[userTasks.length-1]);
+                System.out.println("Now you have " + userTasks.length + " in the list.");
+                break;
+
+
+            default:
+                Task newTask = new Task(userInput);
+                userTasks = AddUserTask(userTasks, newTask);
+                System.out.println("added: " + newTask.getTaskName());
+                System.out.println("Now you have " + userTasks.length + " in the list.");
+                break;
+            }
+        }
+
+
+
+
```

Comment:
Can consider removing extra blank lines.

Reply:
I agree, can consider standardising the empty line spacing between each cases and the extra empty lines to make the code look neater.